.github/workflows/ci.yml

```diff
@@ -312,7 +312,7 @@ jobs:
           submodules: recursive
       - name: Install Doxygen
         run: |
-          wget -qO- "https://sourceforge.net/projects/doxygen/files/rel-1.9.2/doxygen-1.9.2.linux.bin.tar.gz/download" | sudo tar --strip-components=1 -xz -C /usr/local
+          wget -qO- "https://sourceforge.net/projects/doxygen/files/rel-1.9.5/doxygen-1.9.5.linux.bin.tar.gz/download" | sudo tar --strip-components=1 -xz -C /usr/local
```

Comment:
[Ubuntu packages](https://packages.ubuntu.com/jammy/doxygen) is showing an older, ubuntu specific version. Any reason to not use this? Then we could just install with `apt-get`